migrations/0044_sub_issuer_creds.up.sql

```diff
@@ -0,0 +1,3 @@
+-- adding metadata about the credentials (used first for validity period)
+alter table order_creds add column metadata jsonb default null;
+alter table order_creds add column credential_type text default null;
```

Comment:
Why is this text?

Reply:
Mainly so that the migration will not take a long time, text with default null seems to be really fast when provisioning on postgres.